### PR TITLE
allow for initialization of logrus env vars

### PR DESCRIPTION
**Pull request by bparees**

### CODE REVIEW

**Comment by bparees** [comment]
@smarterclayton ptal

File: pkg/serviceability/logrus.go [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+import (
+	"os"
+	"strings"
+
+	"github.com/sirupsen/logrus"
+)
+
+// InitLogrus initializes logrus by setting a loglevel for it.
+func InitLogrus(level string) {
```
Comment (deads2k) [inline pkg/serviceability/logrus.go]: Who else uses this?   I don't see any uses in openshift/origin.  I remember a usage in image-registry I think, but I don't recall a second one.

Comment (bparees) [reply pkg/serviceability/logrus.go]: the library that the imagesignature feature relies on, uses logrus.  those are the logs i'm trying to get enabled w/ this.

**Comment by bparees** [comment]
@deads2k now that you have the context on this do you have any concerns?

**Comment by bparees** [comment]
@deads2k bump

**Comment by deads2k** [comment]
> @deads2k bump

@bparees after the re-rebase to 1.11 GA lands.

**Comment by deads2k** [comment]
lgtm

needs rebase.

**Comment by bparees** [comment]
clean rebase, merging.

**Pull request merged**